src/Components/Instructions/Instructions.tsx

```diff
@@ -0,0 +1,38 @@
+import './Instructions.css';
+
+const Instructions = () => {
+
+  function closePopup() {
+    localStorage.setItem("returningUser", "true");
+    var popup:any = document.getElementById("popup");
```

Comment:
Typically in React we don't use getElementById, we  should be able to just use an onclick attribute.